### PR TITLE
[CHORE] Configure react-cookies for auth

### PR DESCRIPTION
**Pull request by MikeVedsted**
- [X] Implement cookieprovider
- [X] Store auth info in cookie
- [x] Use cookie to retrieve auth info
- [x] Refactor navbar to use info from cookie

### CODE REVIEW

File: client/src/components/Navbar/index.tsx [hunk 1]
```diff
@@ -11,8 +10,7 @@ import './Navbar.scss'
 const Navbar = () => {
   const [navBg, setNavBg] = useState(true)
   const [cookies, setCookie] = useCookies(['x-auth-token'])
-  const decodedToken = jwt.decode(cookies['x-auth-token'])
```
Comment (Rostislav013) [inline client/src/components/Navbar/index.tsx:14]: Do we still need here `'x-auth-token'` or can use `user`? Or does it really need smth in `useCookies()`?

**Review by Rostislav013 — APPROVED** [review]

**Comment by Rostislav013** [comment]
If it works - it works.

**Comment by Rostislav013** [comment]
Also can refactor `EventForm` components. It needs `user_id` too

File: client/src/components/Navbar/index.tsx (hunk 1 above)
Comment (MikeVedsted) [reply client/src/components/Navbar/index.tsx]: not sure how I missed this. It has been corrected now 👍

**Comment by MikeVedsted** [comment]
> Also can refactor `EventForm` components. It needs `user_id` too

Now passing user_id as intended 👍

**Review by MarinaFroes — APPROVED** [review]

**Pull request merged**